bin/index.js

```diff
@@ -118,21 +118,26 @@ function displayFileList() {
     console.log(chalk.bold.cyan("You don't have a manifest folder"));
   } else {
     console.log(chalk.bold.cyan("You current file list: "));
-
     fs.readdirSync(manifestFolder).forEach(file => {
+      if (!validManifestExt(file)) { return; };
```

Comment:
This is a good guard clause to add. Including the error message you've already created [here](https://github.com/layr-team/Layr/pull/43/files#diff-9ea21889e9fbc7d82e7dabb0fb280808R156) telling a user about the problem with their file extension would be helpful.

Reply:
This is a guard to prevent reading other files that user might accidentally add to the folder and we will still try to read it, e.g.: "xxx.txt" which should not be in our list. Do you mean we should add a log to tell users that we only read the manifest files for you?

Reply:
Yeah, in the example you brought up it would be fine to silently skip them. Maybe the more important message is to alert someone if they don't have any `.batchain` files (or they messed up with `.batchain` file extension format) to help them debug. Something like:

```
let manifestCount
fs.readdirSync(manifestFolder).forEach(file => {
  if (!validManifestExt(file)) { return; };
  manifestCount +=1
  ...
}

if (manifestCount === 0) { log that no manifest files were found }
```

What do you think?

Reply:
Sounds good, I will make such modification

Reply:
Logs added: 1395893cc0277b0ca379864e5a260d57abb59ba9